src/msg/msg_types.h

```diff
@@ -442,16 +442,19 @@ struct entity_inst_t {
     return i;
   }
 
-  void encode(bufferlist& bl) const {
+  void encode(bufferlist& bl, uint64_t features) const {
     ::encode(name, bl);
-    ::encode(addr, bl);
+    ::encode(addr, bl, features);
```

Comment:
eventually it will encode differently if the new feature is present.  not 
yet, though.. first step is just to plumb features down to this call site 
everywhere